File: api-stockhub/stockhub/src/main/java/com/api/stockhub/dtos/EquipamentoRecordDto.java
package com.api.stockhub.dtos;

public record EquipamentoRecordDto(Long idEquipamento,String data, String modelo, String armazenamento,
		String memoriaRam, String processador, String office, String nomeMaquina, String numeroDeSerie, String linha,
		 boolean emprestimo,String tipo, String observacao) {

}

File: StockHub/StockHub/Domain/DTOs/EquipamentoRecordDto.cs
namespace StockHubAPI.Dtos
{
    public record EquipamentoRecordDto(
        long IdEquipamento,
        string Data,
        string Modelo,
        string Armazenamento,
        string MemoriaRam,
        string Processador,
        string Office,
        string NomeMaquina,
        string NumeroDeSerie,
        string Linha,
        bool Emprestimo,
        string Tipo,
        string Observacao
    );
}